spec/components/schemas.yaml

```diff
@@ -1,74 +1,75 @@
+openapi: 3.0.1
 components:
   schemas:
-    #assetQuery:
-    #  additionalProperties:
-    #    $ref: "#/components/schemas/assetQueryProp"
-    #  description: >-
-    #    Define which properties of the asset to query and the operations to apply.
-    #
-    #
-    #    The following properties can be queried:
-    #
-    #    - `type`: query for assets with this specific media type
-    #
-    #    - `proj:epsg`: query for assets with this specific epsg
-    #
-    #    - `eo:gsd`: query for assets with this specific gsd
-    #
-    #    - `geoadmin:variant`: query for assets with this specific variant
-    #  example:
-    #    type:
-    #      eq: image/tiff
-    #  type: object
-    #assetQueryFilter:
-    #  description: Allows users to query asset properties for specific values
-    #  properties:
-    #    assetQuery:
-    #      $ref: "#/components/schemas/assetQuery"
-    #  type: object
-    #assetQueryProp:
-    #  anyOf:
-    #    - description: >-
-    #        If the object doesn't contain any of the operators, it is equivalent to using the
-    #        equals operator
-    #    - description: Match using an operator
-    #      properties:
-    #        contains:
-    #          description: >-
-    #            Find items with a property that contains the specified
-    #            literal string, e.g., matches ".*<STRING>.*".
-    #            A case-insensitive comparison must be performed.
-    #          type: string
-    #        endsWith:
-    #          description: >-
-    #            Find items with a property that ends with the specified string. A case-insensitive
-    #            comparison must be performed.
-    #          type: string
-    #        eq:
-    #          description: >-
-    #            Find items with a property that is equal to the specified value. For strings, a
-    #            case-insensitive comparison must be performed.
-    #          nullable: true
-    #          oneOf:
-    #            - type: string
-    #            - type: number
-    #            - type: boolean
-    #        in:
-    #          description: >-
-    #            Find items with a property that equals at least one entry in the specified array.
-    #            A case-insensitive comparison must be performed.
-    #          items:
-    #            oneOf:
-    #              - type: string
-    #              - type: number
-    #          type: array
-    #        startsWith:
-    #          description: >-
-    #            Find items with a property that begins with the specified string. A case-insensitive
-    #            comparison must be performed.
-    #          type: string
-    #      type: object
-    #  description: Apply query operations to a specific property
+    assetQuery:
```

Comment:
Asset's POST search endpoint is not yet implemented, probably keep this commented out until it will be?

Reply:
We don't gain anything by commenting this out, code is still here. However the loose the folding functionality in IDE like vscode when the code is commented. Because this schema is nowhere references it will not be showed anywhere, therefore I prefer keeping it un-commented to have the possibility to fold all schemas with an IDE which make the navigation in code easier.